yet-another-docker-plugin/src/main/java/com/github/kostyasha/yad/DockerCloud.java

```diff
@@ -221,12 +221,13 @@ private DockerSlave provisionWithWait(DockerSlaveTemplate template, Provisioning
         final DockerContainerLifecycle dockerContainerLifecycle = template.getDockerContainerLifecycle();
         final String imageId = dockerContainerLifecycle.getImage();
         final DockerComputerLauncher computerLauncher = template.getLauncher();
+        final String osType = this.getClient().infoCmd().exec().getOsType();
```

Comment:
99% expect ostype to come from image, that will be correct and work without adding additional arguments.